src/main/java/nextstep/subway/path/application/dto/PathSearchType.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.subway.path.application.dto;
+
+public enum PathSearchType {
+    DISTANCE,
+    DURATION;
+
+    public boolean isDuration() {
+        return this.equals(DURATION);
```

Comment:
Nit: 열거형 상수는 싱글턴 인스턴스이기 때문에, `==`으로 비교할 수도 있습니다.
```suggestion
        return this == DURATION;
```